src/components/GenQRCode.vue

```diff
@@ -37,6 +37,20 @@ export default {
       _procError(err)
     })
   },
+  watch: {
+    qrcode(newQRCode, oldQRCode) {
```

Comment:
should put this into a computed so in case then qrcode change, the QR Code image change, no more duplicated code